README.md

```diff
@@ -29,7 +29,7 @@ It will expose the following endpoints (both over https):
 ```
 * /incoming-data-app/multipartMessageBodyBinary to receive data (MultiPartMessage) with binary body from Data App (the A endpoint in the above picture)
 * /incoming-data-app/multipartMessageBodyFormData to receive data (MultiPartMessage) with form-data body from Data App (the A endpoint in the above picture)
```

Comment:
Can you add also httpHeader A-endpoint here also, looks like we forgot to add it in README :(

Reply:
On my way :)